druid/examples/layout.rs

```diff
@@ -60,6 +60,17 @@ fn build_app() -> impl Widget<u32> {
             weight,
         );
     }
+
+    // aspect ratio box
+    let aspect_ratio_label = Label::new("Hello there, this box it's aspect-ratio. Notice text will overflow if box becomes too small.")
```

Comment:
```suggestion
    let aspect_ratio_label = Label::new("Hello there, this is an aspect-ratio box. Notice how the text will overflow if the box becomes too small.")
```

Reply:
Haha sorry, I did not proof read that at all.

Reply:
No worries 😄 Feel free to just replace my suggestion with your own wording if you want